index.html

```diff
@@ -188,44 +195,45 @@ <h3 class="mt-post">Multi-Post Stories</h3>
         </div>
         <div id="section-5" class="contain5">
           <div class="s5">
-            <div class="about-text">
-              <div class="head-text">
-                <h1>About Myself</h1>
+            <div id="about1">
+              <div class="about-text">
+                <div class="head-text">
+                  <h1>About Myself</h1>
+                </div>
+                <p>
+                  Hello I’m a software developer! I can help you build a product
+                  , feature or website Look through some of my work and
+                  experience! If you like what you see and have a project you
+                  need coded, don’t hestiate to contact me.
+                </p>
               </div>
-              <p>
-                Hello I’m a software developer! I can help you build a product ,
-                feature or website Look through some of my work and experience!
-                If you like what you see and have a project you need coded,
-                don’t hestiate to contact me.
-              </p>
-            </div>
-            <h2>LET'S CONNECT</h2>
+              <h2>LET'S CONNECT</h2>
 
-            <div class="base-icon-5">
-              <ul>
-                <li>
-                  <a href=""><img src="images/Vector-1.svg" alt="image" /></a>
-                </li>
-                <li>
-                  <a href=""
-                    ><img src="images/Linkedinicon.svg" alt="image"
-                  /></a>
-                </li>
-                <li>
-                  <a href=""><img src="images/Vector-2.svg" alt="image" /></a>
-                </li>
-                <li>
-                  <a href=""><img src="images/Vector-3.svg" alt="image" /></a>
-                </li>
-                <li>
-                  <a href=""><img src="images/Vector.svg" alt="image" /></a>
-                </li>
-              </ul>
-            </div>
-            <div class="resume">
-              <a href="">Get my resume</a>
+              <div class="base-icon-5">
+                <ul>
+                  <li>
+                    <a href=""><img src="images/Vector-1.svg" alt="image" /></a>
+                  </li>
+                  <li>
+                    <a href=""
+                      ><img src="images/Linkedinicon.svg" alt="image"
+                    /></a>
+                  </li>
+                  <li>
+                    <a href=""><img src="images/Vector-2.svg" alt="image" /></a>
+                  </li>
+                  <li>
+                    <a href=""><img src="images/Vector-3.svg" alt="image" /></a>
+                  </li>
+                  <li>
+                    <a href=""><img src="images/Vector.svg" alt="image" /></a>
+                  </li>
+                </ul>
+              </div>
+              <div class="resume">
+                <a href="">Get my resume</a>
+              </div>
```

Comment:
- [ ] Kindly ensure that your Get my Resume link is  aligned properly with the about content vertically. You can refer to the images below for guidance
> Your Design
![image](https://user-images.githubusercontent.com/90556221/178105691-8b85b6d0-f4b6-40e4-bc38-3fc6ba6649cc.png)

> Figma Design
![image](https://user-images.githubusercontent.com/90556221/178105709-eabf44c4-451d-4f7c-976a-3608395b9c02.png)

- [ ] Please ensure that the About Myself heading has the correct font weight as can be seen above.